server/resource_server/controller/controller.go

```diff
@@ -320,3 +320,19 @@ func GetUsageStats(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func CheckBackendURI(w http.ResponseWriter, r *http.Request) {
```

Comment:
you haven't add this handler to main.go for routing, how frontend even able to access this api?

Reply:
I just removed before pushing. I added!